### DISCUSSION TITLE
[Feature Request] <zksync Era official bridge withdrawal issue>

### DISCUSSION THREAD

**Discussion by Eric-ett** [zkSync-Community-Hub/zksync-developers#636] [General] [answered]
### Environment

Testnet

### Provide a brief description of the functionality you're trying to implement and the issue you are running into.

Three days ago, I withdrew an amount of ETH from zksync Era (https://era.zksync.network/tx/0xd58594cf28ee73dcc49d5edd38421d7cd9a53726e891ebb4c8609502653503e3), but on the claim page I could not click the claim button, but "Fee estimation error: The contract function " appeared. finalizeEthWithdrawal" reverted with the following reason:
xx

Contract Call:
address: 0x32400084c286cf3e17e7b677ea9583e60a000324..." error,
And asked me to try again, but after trying again, I kept getting this error and couldn't claim funds.
I want to know what the problem is and how can I claim the funds

### Additional Details

Another display problem is that some withdraw transactions that have been claimed on the https://portal.zksync.io/transfers page are still displayed as "bridging", and when you click "go to claim" to enter the "claim withdraw" page will prompt "Fee estimation error: The contract function "finalizeEthWithdrawal" reverted with the following reason:
Withdrawal is already finalized" error.

**Comment by alhenaai** [comment]
It seems you are encountering a known issue with the "finalizeEthWithdrawal" function on the ZKsync Era official bridge. This error typically occurs due to issues with fee estimation or because the withdrawal process has already been finalized but hasn't updated correctly on the user interface.

Here are a few steps you can take to resolve this issue:

1. **Clear Browser Cache**: Sometimes, clearing your browser's cache can resolve display issues and errors related to outdated information being held. Please clear your browser cache and try the claim process again.

2. **Use an Alternative Interface**: If the official bridge page continues to show errors, you might want to try using an alternative interface like [SyncSwap](https://syncswap.xyz/move) to finalize your transaction. Just enter your transaction details there and look for a 'claim' button.

3. **Check Etherscan**: Verify if the withdrawal has been finalized by checking the transaction on [Etherscan](https://etherscan.io/). You can search using the transaction hash you provided and look for any internal transactions indicating that the funds have been moved.

4. **Contact Support**: If none of the above solutions work, it might be helpful to reach out directly for support. Since you are experiencing this on the testnet, it could be related to ongoing updates or issues specific to the test environment.

For the issue with transactions still displayed as "bridging," this is likely a UI/UX bug where the interface does not refresh the status promptly. Reporting these details as feedback on the testnet can help improve the system.

If you need further assistance, please let me know!

Your feedback helps us serve you better. If this fully addresses your question, please give it a thumbs up. If it doesn't, please give it a thumbs down. This will inform the community and team so we can improve our support.

**Reply by bxpana** [reply]
This type of error typically means that the withdrawal was already finalized, but I see that it's not showing on L1. Let me check on that.

**Reply by bxpana** [reply]
Did you just try to claim them again on the portal or did you use an alternative way to claim?

**Reply by Eric-ett** [reply, marked as answer]
I used SyncSwap and manually entered the withdrawal hash, and successfully claimed funds on L1.

**Reply by bxpana** [reply]
Good to know and will have the team check out what's going on with the Portal then.
